sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -121,8 +135,11 @@ export class IdentityClient extends ServiceClient {
       });
 
       const response = await this.sendTokenRequest(webResource, expiresOnParser);
+      logger.info(`refreshAccessToken: client ID: ${clientId} token response received`);
       return response;
     } catch (err) {
+      logger.error(`refreshAccessToken: client ID: ${clientId} failed, ${err}`);
```

Comment:
At this point we don't know yet whether it truly failed, it could just be that AAD needs the user to reauthenticate.  I'd move this line into the `else` below.  You might log in the `if` below that the access token could not be refreshed and that the user will be reauthenticated.

Reply:
I split it up a bit more now that I get what's happening and now there are two log messages for the different code paths.